envoy/api/v2/core/protocol.proto

```diff
@@ -33,6 +33,20 @@ message Http1ProtocolOptions {
   // envoy as their HTTP proxy. In Unix, for example, this is typically done by setting the
   // *http_proxy* environment variable.
   google.protobuf.BoolValue allow_absolute_url = 1;
+
+  // [#not-implemented-hide:]
+  // Handle incoming HTTP/1.0 requests.
+  // This is off by default, and not fully HTTP/1.0 compliant. There is planned support for HTTP/1.0
+  // style connect logic, dechunking, and handling lack of client host iff the default_host field is
+  // configured.
+  // Any requests for further 1.0 compatibility can be added here:
+  // https://github.com/envoyproxy/envoy/issues/170
+  bool accept_http_10 = 2;
+
+  // [#not-implemented-hide:]
+  // A default host. This is highly suggested if accept_http_10 is true on as Envoy does not
```

Comment:
Should we potentially only support this option IFF accept_http_10 is true?

Reply:
I don't think so.  We don't have any other HTTP1 config which is gated on things, and I think internally we're going to want a default host for 1.1 because it turns out not everyone adheres to the spec :-(

Reply:
OK fair enough. Can we add more documentation to the effect of this will also enable for HTTP/1.1 which is not standards compliant behavior?

Reply:
Sure thing!